dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/teradata/TeradataLogsConnector.java

```diff
@@ -85,11 +78,8 @@ public class TeradataLogsConnector extends AbstractTeradataConnector implements
 
     private static final Logger LOG = LoggerFactory.getLogger(TeradataLogsConnector.class);
 
-    protected static final DateTimeFormatter SQL_FORMAT = DateTimeFormatter.ISO_OFFSET_DATE_TIME.withZone(ZoneOffset.UTC);
-    @VisibleForTesting
-    /* pp */ static final String DEF_LOG_TABLE = "dbc.DBQLogTbl";
     @VisibleForTesting
-    /* pp */ static final String DEF_QUERY_TABLE = "dbc.DBQLSQLTbl";
+    public static String EXPRESSION_VALIDITY_QUERY = "SELECT TOP 1 %s FROM %s";
```

Comment:
TeradataLogsConnector is in the same package. The visibility can be package private.